packages/sonarwhal/src/lib/cli/init.ts

```diff
@@ -180,18 +182,27 @@ export const initSonarwhalrc = async (options: CLIOptions): Promise<boolean> =>
 
     const initialAnswer: inquirer.Answers = await inquirer.prompt(initialQuestion);
 
-    const sonarwhalConfig = initialAnswer.configType === 'predefined' ?
+    const result = initialAnswer.configType === 'predefined' ?
         await extendConfig() :
         await customConfig();
 
-    // No resources installed or no configuration packages available, we have to quit
-    if (!sonarwhalConfig) {
-        return true;
-    }
-
     const filePath: string = path.join(process.cwd(), '.sonarwhalrc');
 
-    await promisify(fs.writeFile)(filePath, JSON.stringify(sonarwhalConfig, null, 4), 'utf8');
+    await promisify(fs.writeFile)(filePath, JSON.stringify(result.config, null, 4), 'utf8');
+
+    if (result.packages) {
```

Comment:
Maybe do a check like:

```js
if(Array.isArray(result.packages) && result.packages.length > 0) { 
  // magic
}
```

Right now it looks like in order to exist it will always be an array with items but we don't know what will have in the future.